Website Style Upgrade

LegendaryLinux opened the pull request:
## Changelog
- Themes added to AutoWorld class
- Themes added to WebHost
- Accessibility upgrades
- New fonts
- New backgrounds
- New styles

Berserker66 commented on worlds/AutoWorld.py:
```diff
@@ -132,6 +136,10 @@ class World(metaclass=AutoWorldRegister):
     item_names: Set[str]  # set of all potential item names
     location_names: Set[str]  # set of all potential location names
 
+    # If the game displays all contained items to the user, this flag pre-fills the hint system with this information
```
this is gone. PR should not re-add it.

LegendaryLinux replied:
Please be more specific with your feedback. What is "this"?

Berserker66 replied:
The 4 lines this PR is adding at this marker point. sending_visible is no longer a thing.

LegendaryLinux replied:
Those lines have been removed. I'm not actually sure how they snuck back in there, considering I never touched them while working on the WebHost.

Berserker66 commented on WebHostLib/__init__.py:
```diff
@@ -70,6 +70,13 @@ def to_url(self, value):
 
 
+def get_world_theme(game_name: str):
+    for game, world in AutoWorldRegister.world_types.items():
```
You can just lookup instead of iterating; `AutoWorldRegister.world_types[game_name].web.theme`

LegendaryLinux replied:
This one has been fixed and pushed.

Berserker66 commented:
I guess last question then, squash or rebase?

LegendaryLinux commented:
Just squash it, I think.

LegendaryLinux commented:
Last two commits removed references to the Z5Client from the OoT English tutorial and fixed an overflow problem on code blocks with very long lines. Should have also fixed that merge conflict.

Ijwu commented:
> Just squash it, I think.

K, squashing. Great job on this, it looks amazing. Big credit to our designer, too.
